modules/apps/jaggery/emm/src/emm/units/appbar/public/js/appbar.js

```diff
@@ -77,7 +77,7 @@ function loadNotifications(){
     var serviceURL = "/mdm-admin/notifications/NEW";
 
     var successCallback = function (data) {
-        if(data){
+        if(!data){
             data = "[]";
```

Comment:
If there are no json payload for data  value,we checked and give empty array,otherwise we can not parse to json.